src/test/matching_test.cpp

```diff
@@ -70,26 +74,14 @@ TEST(Matching, StaticTest_2) {
     Nodes nodes;
 
     size_t arity = 7;
-    order.addSearch(SearchSignature(arity, 121));
-    nodes.insert(SearchSignature(arity, 121));
-    order.addSearch(SearchSignature(arity, 104));
-    nodes.insert(SearchSignature(arity, 104));
-    order.addSearch(SearchSignature(arity, 53));
-    nodes.insert(SearchSignature(arity, 53));
-    order.addSearch(SearchSignature(arity, 49));
-    nodes.insert(SearchSignature(arity, 49));
-    order.addSearch(SearchSignature(arity, 39));
-    nodes.insert(SearchSignature(arity, 39));
-    order.addSearch(SearchSignature(arity, 33));
-    nodes.insert(SearchSignature(arity, 33));
-    order.addSearch(SearchSignature(arity, 32));
-    nodes.insert(SearchSignature(arity, 32));
-    order.addSearch(SearchSignature(arity, 23));
-    nodes.insert(SearchSignature(arity, 23));
-    order.addSearch(SearchSignature(arity, 11));
-    nodes.insert(SearchSignature(arity, 11));
-    order.addSearch(SearchSignature(arity, 7));
-    nodes.insert(SearchSignature(arity, 7));
+
+    std::array<uint64_t, 10> patterns = {7, 11, 23, 32, 33, 39, 49, 53, 104, 121};
```

Comment:
you can shorten this (and other instances of this) to `uint64_t patterns[] { .... };`

Reply:
Done